mirascope/base/types.py

```diff
@@ -130,6 +130,7 @@ class BaseCallResponse(BaseModel, Generic[ResponseT, BaseToolT], ABC):
     tool_types: Optional[list[Type[BaseToolT]]] = None
     start_time: float  # The start time of the completion in ms
     end_time: float  # The end time of the completion in ms
+    cost: Optional[float] = None  # The cost of the completion in dollars
```

Comment:
is this optional because of Gemini? I would rather this be required and then just be `0.0` for gemini since we'll note it isn't supported in the docs

Reply:
no

Reply:
Note: sometimes usage is none.